frontends/web/src/app.tsx

```diff
@@ -249,5 +237,14 @@ class App extends Component<Props, State> {
     }
 }
 
-const HOC = translate()(App);
+const subscribeHOC = subscribe<SubscribedProps, TranslateProps>(
+    {
+        accounts: 'accounts',
+        devices: 'devices/registered',
+    },
+    true,
+    false,
```

Comment:
Technically I don't think you need the true and false here since they're the default values for those params?

Reply:
good point, i missed that

Reply:
Chose to not do it anyway to save some time, can't hurt to be explicit either.